test/models/auto_complete_test.rb

```diff
@@ -4,6 +4,7 @@
 
 class AutoCompleteMock < AutoCompleteByString
   attr_accessor :rough_matches, :limit
+
```

Comment:
Fixes offense of cop added by RuboCop 0.83 (Layout/EmptyLinesAroundAttributeAccessor).
See https://github.com/JoeCohen/mushroom-observer/commit/36d4982